### ISSUE TITLE
Enable md2jira markdown initialization

### ISSUE THREAD

**Issue by TristanCacqueray** [ButlerOS/haskell-jira-client#3]
As a first step toward two way synchronizations, md2jira could feature an initialization command to populate an empty file using the search API.

**Comment by apevec** [comment]
@TristanCacqueray before md2jira can do 2way sync, could md2jira insert as a first line in Jira description something like:
*** This description is autogenerated by md2jira from source <link to
md file in a repo> ***

so that we know where to update the source of truth and to avoid
information loss?

For an example where description change was lost see in RHOSZUUL-2200 History changes - 2025/02/14 1:43 PM

**Comment by TristanCacqueray** [comment]
@apevec good idea, thanks! I added the header in version 0.2: https://github.com/ButlerOS/haskell-jira-client/pkgs/container/haskell-jira-client/358596220?tag=0.2
